backend/isoko-backend/src/handlers/business/post-list-business.js

```diff
@@ -14,83 +19,106 @@ const hash = (name, street) => {
 
-   let params;
+   let businessId;
 
+   // Required fields
    const name = _.get(requestBody, 'name');
    const street = _.get(requestBody, 'street');
    const onlineUrl = _.get(requestBody, ['links', 'BusinessURL']);
+   const state = _.get(requestBody, 'state');
+   const city = _.get(requestBody, 'city');
 
    // Check for required fields for hashing
-   if (type == 'B&M') {
-      if (name && street) {
-         params = {
-            TableName: BUSINESS_TABLE,
-            Item: {
-               pk: hash(name, street),
-               sk: 'INFO',
-               ...requestBody,
-            },
-         };
+   if (type == BRICK_AND_MORTAR) {
+      if (name && street && state && city) {
+         businessId = hash(name, street);
       } else {
-         return {
-            statusCode: 400,
-            body: {
-               error: 'name and street are required fields for Brick and Mortar Businesses.',
-            },
-         };
+         return get400Response(
+            'Required field not present, ensure name, street, state, and city are present for Brick and Mortar Businesses.'
+         );
       }
-   } else if (type == 'Online') {
+   } else if (type == ONLINE) {
       if (name && onlineUrl) {
-         params = {
-            TableName: BUSINESS_TABLE,
-            Item: {
-               pk: hash(name, onlineUrl),
-               sk: 'INFO',
-               ...requestBody,
-            },
-         };
+         businessId = hash(name, onlineUrl);
       } else {
-         return {
-            statusCode: 400,
-            body: {
-               error: 'name and links.BusinessURL are required fields for Online businesses.',
-            },
-         };
+         return get400Response(
+            'Required field not present, ensure name and links.BusinessURL are present for Online businesses.'
+         );
       }
+   } else {
+      return get400Response(`Invalid business type: ${type}`);
    }
 
    let response;
 
    try {
-      console.info(`DynamoDB put params: ${params}`);
-      await docClient.put(params).promise();
+      const searchResultsParams = {
+         TableName: SEARCH_RESULTS_TABLE,
+         Item: {
+            pk: type == BRICK_AND_MORTAR ? `${state}#${city}` : ONLINE,
+            sk: `${category}#${businessId}`,
+            name,
+            city,
+            type,
+            category,
+            tags: _.get(requestBody, 'tags', []),
+            keywords: _.get(requestBody, 'keywords', []),
+            stars: 0,
+            numReviews: 0,
+            shortDesc: _.get(requestBody, 'shortDesc'),
+            verified: _.get(requestBody, 'verified', false),
+            businessId,
+            photo: _.get(requestBody, 'photoLink'),
+            hours: _.get(requestBody, 'hours', {}),
+            timestamp: _.get(requestBody, 'timestamp'),
+         },
+      };
+      console.info(
+         `DynamoDB put params for SearchResults Table: ${searchResultsParams}`
+      );
+
```

Comment:
not sure why but this is what's printing to the console
`DynamoDB put params for SearchResults Table: [object Object]`
so unless there's a way to get it to log the object properly i'd say we prob don't need it